grakn-engine/src/main/java/ai/grakn/engine/loader/MutatorTask.java

```diff
@@ -96,8 +97,8 @@ private boolean insertQueriesInOneTransaction(GraknTx graph, Collection<Query> i
                     addTask(PostProcessingTask.createTask(this.getClass(), engineConfiguration()
```

Comment:
Actually, I just realised we can delete the MutatorTask. Maybe for another PR

Reply:
I will give it a try. I like deleting code 🙂

Reply:
On snd thought I will leave it for another PR. Ripping it out will affect a lot it seems.